packages/vision/src/index.js

```diff
@@ -410,7 +412,9 @@ Vision.prototype.detect = function(images, options, callback) {
 
       var detections = foundImages
         .map(groupDetectionsByImage)
-        .map(removeExtraneousAnnotationObjects)
+        .map(function(image) {
+          return curation.removeExtraneousAnnotations(image, options.types);
```

Comment:
I am confused about this code. Are these _images_ or _annotations_? The variable is named `foundImages` but the old method expected _annotations_. Is `image` the right thing for me to be sending here, or should I be sending an annotations property?

The unit tests still pass with this code change, but I am not sure the code change is _actually_ correct.